src/niivue.js

```diff
@@ -1027,13 +1035,18 @@ Niivue.prototype.touchEndListener = function (e) {
 // not included in public docs
 // handler for mouse move over canvas
 // note: no test yet
+/**
+ * Handler for move move event
+ * @param {MouseEvent} e mouse move event
+ */
 Niivue.prototype.mouseMoveListener = async function (e) {
```

Comment:
Can we keep the original code flow of in this function and only add behavior in case the **enableBorderHighlight** setting is set to true?

Reply:
Okay yes I have done it, so the update would only happen if the option is enabled. Is that the way you meant it?